src/controllers/hid/hidcontroller.h

```diff
@@ -8,7 +8,7 @@
 #ifndef HIDCONTROLLER_H
 #define HIDCONTROLLER_H
 
-#include <hidapi.h>
+#include <hidapi/hidapi.h>
```

Comment:
It builds on my Trusty system with system hidapi and on Travise Precise which has none. 

For the internal case we have both include paths: 
build.env.Append(CPPPATH=[self.HIDAPI_INTERNAL_PATH])
build.env.Append(CPPPATH=[os.path.join(self.HIDAPI_INTERNAL_PATH, 'hidapi')])

Is it broken for you?